packages/build-tools/src/ios/xcpretty.ts

```diff
@@ -66,4 +72,18 @@ export class XcodeBuildLogger {
       }
     }
   }
+
+  private async findBundlerErrors(logsPath: string): Promise<void> {
+    try {
+      const logFile = await fs.readFile(logsPath, 'utf-8');
+      const match = logFile.match(
+        /Welcome to Metro!\s* Fast - Scalable - Integrated\s*([\s\S]*)Run CLI with --verbose flag for more details.\nCommand PhaseScriptExecution failed with a nonzero exit code/
+      );
+      if (match) {
+        this.logger.info(match[1]);
+      }
+    } catch (err) {
+      this.logger.error({ err }, 'Failed to read xcode logs');
```

Comment:
Is this logline user-facing? If so, I'd skip it.

Reply:
yes it's user-facing
it's unlikely that this error would happen, that file exists because xcpretty already reads from it, but I don't want to leave it with empty catch, because in case of any problems we wouldn't know about there was an issue